src/js/libcs/OpDrawing.js

```diff
@@ -107,18 +107,26 @@ evaluator.fillcircle$2 = function(args, modifs) {
 };
 
 evaluator.drawarc$3 = function(args, modifs) {
-    return eval_helper.drawarc(args, modifs, "D");
+    var newargs = [csgeo.csnames[args[0].name].homog,
+        csgeo.csnames[args[1].name].homog,
+        csgeo.csnames[args[2].name].homog
+    ];
+    return eval_helper.drawarc(newargs, modifs, "D");
 };
 
 evaluator.fillarc$3 = function(args, modifs) {
-    return eval_helper.drawarc(args, modifs, "F");
+    var newargs = [csgeo.csnames[args[0].name].homog,
+        csgeo.csnames[args[1].name].homog,
+        csgeo.csnames[args[2].name].homog
+    ];
+    return eval_helper.drawarc(newargs, modifs, "F");
 };
 
```

Comment:
To avoid this problem from manifesting itself again, could you please change the existing arc examples to (also?) include some arc drawn using vectors? E.g. make it `drawarc(A, B.homog, C.xy)` or something like that.